### ISSUE TITLE
zowe --help-web on Linux launches help in text editor not browser

### ISSUE THREAD

**Issue by gejohnston** [zowe/imperative#253]
On Linux, when the zowe --help-web command must generate the web help, zowe displays the web-help/index.html file in a text editor instead of in a browser. 

Subsequent zowe --help-web commands work correctly.

This behavior was first noticed on July 11, 2019.

**Comment by gejohnston** [comment]
By testing old snapshots, I identified the point in time when this behavior was introduced.
The browser launched correctly in the commit :
- bd0d817dcc64aa9c5d014c8675ebebf99eb07d7a = Enable the zowe command to return to a command prompt on Linux while still displaying the HTML help.

The text editor was launched instead of the browser in the following commit :
- 2b21f2caff88c90b1af3bd8a739907c504d5c81a = Bundle node modules for web help with browserify

We will have to identify a means to correct this misbehavior when using Browserify, or remove that implementation.

Additional debugging information:
- In WebHelpManager, right before the call to the opener() utility, I created a loop that checked the size of the index.html file every 5 seconds up to 10 times (a duration of 50 seconds). Each time, the file size was zero.
- I believe that because the file is empty, the heuristics that Linux uses to detect the file type, decides it much launch a text editor instead of a browser.
- As soon as the zowe command finishes, the index.html file contains its correct contents.
    - I assume that is why the separately-spawned text editor shows correct HTML text in its window.
    - That would also explain why a second 'zowe --help-web' command (which does not have to generate the HTML) correctly displays the help in a browser.

It appears that on Linux the I/O related to the Browserify update does not complete until the zowe command terminates. Perhaps its I/O buffers are not flushed? Perhaps some asynchronous operation is run as if it were synchronous?

**Comment by tjohnsonBCM** [comment]
After doing some testing in a Linux VM, I believe I have identified the issue. I replaced this code for copying files:
`fs.createReadStream(sourceFile).pipe(fs.createWriteStream(destFile));`

with this using the `fs-extra` package:
`fs.copySync(sourceFile, destFile)`

I'll push a commit with this change since it seems to fix the issue. I believe the file metadata wasn't getting copied over previously, so it took a moment after copying for the OS to detect the new file as HTML.
